src/app/components/videochat/jh-video-chat.directive.js

```diff
@@ -122,7 +122,11 @@
           }
           // If current one is still speaking, there is no need to change
           if ( !(current && current.getSpeaking()) ) {
```

Comment:
question is what happen as current is also Speaking, so it ends up with findMain, which in the end is publisher, as current cannot disable video ( from testing his video is still open and only last picture is frozen )